src/containers/guest-home-page/forgot-password/ForgotPassword.jsx

```diff
@@ -94,7 +94,7 @@ const ForgotPassword = () => {
           onChange={handleInputChange('email')}
           required
           size='large'
-          sx={{ mb: '5px' }}
+          sx={{ mb: '5px', mt: '15px' }}
```

Comment:
It seems that according to the design, the margin-top should be about 30px.

Reply:
Yes, my bad, it's 25, not 15

Reply:
I see 32px
![image](https://github.com/Space2Study-UA-1125/frontEnd/assets/126193367/4f658f53-936e-4043-84ec-147e198478ea)

Reply:
![image](https://github.com/Space2Study-UA-1125/frontEnd/assets/93076661/f48453f5-2c37-4e90-8513-a314ad97bf90)

Reply:
![image](https://github.com/Space2Study-UA-1125/frontEnd/assets/93076661/b23c9784-eebc-4e34-a03d-1eab666b68d7)


Here it is even 36px.